app/api/__init__.py

```diff
@@ -1,5 +1,5 @@
 # type: ignore
-from __future__ import annotations
+# isort: dont-add-imports
```

Comment:
Know if there's a way to automagically handle `__init__.py` files globally in the `.isort.cfg`?

Reply:
seems like there is no way to do this sadly